wiki/release.md

```diff
@@ -0,0 +1,30 @@
+# Releases
+
+## Release workflow
+
+the flow should be goes as following
+
+1. Each release is developed against branch at the beginning and this is branched from development `i.e development-2.0`
```

Comment:
Each release is developed on a branch that is derived from the development branch (e.g. development-2.0).